Terms of Reference.md

```diff
@@ -0,0 +1,179 @@
+Considerations in this regard include the reasons why an individual is deemed
+marginalized or an outlier.  A possible reason, or example, is that they were
+either not part of the training data or were poorly represented within it.
+Another reason might be the nature of the model or algorithm(s) that the AI uses
+in making its decisions.  That model or algorithm might produce discriminatory
+results in the context of a marginalized individual and that would indicate it
+is inappropriate to use in this particular case.
+
+The committee will also delineate the kinds of generative ML-AI that are
+relevant to the specification.  For example, an AI that is definitely
+out-of-scope is one that generates photorealistic images of human faces. This
+kind of AI is not properly described as making any decisions. However, a large
+language model (LLM) may or may be in scope. For example, consider this
+scenario: An AI decision tool is known to discriminate and it uses an LLM during
+its processing to arrive at a decision.  The LLM might be the source of the
+discrimination; hence, the LLM is in scope in this scenario.
+
+The committee will develop and document techniques for evaluating an ML-AI in
+order to determine whether it discriminates or can potentially discriminate.
+This might involve the methods used in the field of explainable AI. The
+techniques that are developed by the technical committee will allow an operator
+to judge whether an AI can be trusted when used in a specific context.
+
+The above are the main objectives of the committee.  The committee, in its work,
+may discover other related and relevant objectives, and can include and develop
+them as part of the specification.
+
+Administrative issues that the technical committee will handle include:
+
+- [Meeting Schedule](#meeting-schedule)
+  - The technical committee shall decide among themselves when and how they will
+    conduct their meetings.
+- [Meeting Methodology and Collaborative Processes](#meeting-methodology-and-collaborative-processes)
+  - The committee members shall decide what tools and processes they will use
+    to communicate with one another and to produce drafts of the specification.
+- [Roles and Responsibilities](#role-and-responsibilities)
+  - The committee members shall discuss and decide their roles and
+    responsibilities.
+
+- Includes Technical Experts who will draft the Technical Specification
+  - Consider gender responsiveness when drafting the Technical Specification
+  - Consider environmental responsiveness when drafting the Technical 
+    Specification
+  - Respond to questions/comments during review of Technical Specification
+  - Technical Experts will provide guidance on whether a public or peer review
+    of the Technical Specification is required
+- Vote on publication of Technical Specification
+- Help review Technical Specification, including providing feedback on the 
+  Technical Specification, and facilitating the review process.
```

Comment:
This section was just reference for responsibilities pulled from the statement of work. They should be incorporated as necessary in this document, where appropriate.

Reply:
I have taken a stab at turning some of these into a paragraph and leaving them in the "Purpose of the Technical Committee" section.  But, I don't understand either of the "Consider gender/environmental responsiveness when drafting the Technical Specification" requirements.  If anyone has ideas, feel free to suggest text and where to put it.

Reply:
I'm not sure what is required for environmental responsiveness. 

The gender responsiveness seems to be related to the composition of the technical committee with regards to gender. What's the ratio, what measures were taken to encourage diversity.